simulation_parameters/microbe_stage/organelles.json

```diff
@@ -574,7 +574,7 @@
     "Processes": {},
     "Components": {
       "Storage": {
-        "Capacity": 8
+        "Capacity": 2
```

Comment:
Doesn't this make vacuole feel like almost totally useless organelle as it adds just 2 storage? I feel like it should be a satisfying organelle to place as a solution for not having enough storage.